Simple-User-Management-API/Simple-User-Management-API/Startup.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.HttpsPolicy;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace Simple_User_Management_API
+{
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
```

Comment:
Enable CORS to allow FE access the API.

https://docs.microsoft.com/en-us/aspnet/core/security/cors?view=aspnetcore-3.1